app/assets/stylesheets/application.scss

```diff
@@ -45,6 +45,14 @@ h1.article {
   color: #B7B7B7;
 }
 
+h1.ranking {
```

Comment:
Consider replacing all the h1.ranking, etc. css declarations that use `color: #B7B7B7` with a single:

```css
h1 {
  color: #B7B7B7
}
```

Then keep the exceptions like `h1.post` which uses a different color.